src/data.h

```diff
@@ -9,7 +9,7 @@ typedef ap_int<5>  vtx_t;
 typedef ap_uint<2>  particleid_t;
 typedef ap_int<10> z0_t;  // 40cm / 0.1
 		
-enum PID { PID_Charged=0, PID_Neutral=1, PID_Photon=2, PID_Electron=3 };
+enum PID { PID_Charged=0, PID_Neutral=1, PID_Photon=2, PID_Electron=3, PID_Muon=4 };
```

Comment:
you should also change `particleid_t` to be `ap_uint<3>` since otherwise it can't store the value `4` (sorry I didn't notice it yesterday)